ops/ansible/roles/bfd-server/templates/bfd-server.sh.j2

```diff
@@ -63,10 +63,7 @@ export NEW_RELIC_METRIC_PERIOD='{{ data_server_new_relic_metric_period }}'
 {% endif %}
 
 # Begin smoke testing startup routine
-STARTUP_TESTING_RETRIES='3'
-STARTUP_TESTING_BOOT_TIMEOUT='10' # TODO: temporary reduction; to be fixed/removed in BFD-1813
 STARTUP_TESTING_REQ_TIMEOUT='15'
-STARTUP_TESTING_REQ_BACKOFF_TIMEOUT='10'
```

Comment:
You might consider reviving one or more of these to give the server a little breathing room when it's on its way to booting. This currently results in several sub-second retries without a backoff or grace period.

Reply:
Right, I set up this to 5 seconds